Dockerfile

```diff
@@ -32,8 +32,17 @@ RUN apt update -qq && apt install -qq -y --no-install-recommends \
         ruby-full \
         openssh-client \
         unzip \
+        libssl-dev \
+        libreadline-dev \
+        zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*;
 
+# install ruby 2.4.9 for Fastlane usage
+RUN curl -L https://github.com/sstephenson/ruby-build/archive/v20191205.tar.gz | tar -zxvf - -C /tmp/ \
+  && cd /tmp/ruby-build-* && ./install.sh && cd / \
+  && ruby-build -v 2.4.9 /usr/local && rm -rfv /tmp/ruby-build-* \
```

Comment:
I think just install the latest ruby version is okay.

Reply:
There is no "latest" argument for ruby-build that I can see but if you mean it should be 2.6.5 I can change it to that. I tried using ruby-install first but I wasn't able to replace system Ruby with it.

Reply:
Maybe remove `ruby-full` from install and try ruby-install again ?

Reply:
Ah, it does seem to work now! I needed to add some dependencies. Does this look alright?
```
RUN apt-get update -qq \
  && apt-get install -qq -y --no-install-recommends \
          bison \
          zlib1g-dev \
          libyaml-dev \
          libssl-dev \
          libgdbm-dev \
          libreadline-dev \
          libncurses5-dev \
          libffi-dev \
  && curl -L https://github.com/postmodern/ruby-install/archive/v0.7.0.tar.gz | tar -zxvf - -C /tmp/ \
  && cd /tmp/ruby-install-* \
  && make install \
  && ruby-install --latest --system --cleanup ruby \
  && gem install bundler -N \
  && rm -rf /var/lib/apt/lists/*
```

Reply:
Should I make a new branch since this one has 2.4.9 in the name?